### DISCUSSION TITLE
6/14(화) 제네릭을 활용한 상속

### DISCUSSION THREAD

**Discussion by jjunhaa0211** [Info-iOS/iOStudy#186] [General]
collectionView를 만드는데  collectionView와 CollectionViewCell을 그냥 만드시지 말고 두개 다 상속을 활용해서 만드시면 됩니다. 

조건
- collectionView와 collectionViewCell 전부 제네릭을 활용한 baseCode를 만듭니다.
- collectionView에서는 이미지, 라벨이 있어야합니다.

다음주 화요일까지 제출해주세요

**Comment by Eunho0922** [comment]
넵

**Comment by jjunhaa0211** [comment]
내일까지 제출 입니다 😎
